docs/modules/commons/pages/cli.adoc

```diff
@@ -0,0 +1,298 @@
+= CLI
+
```

Comment:
it's needed to document:
- what is task (because the word is used below)
- general format of the tasks
- where tasks are available